lgc/disassembler/Disassembler.cpp

```diff
@@ -389,49 +440,216 @@ void ObjDisassembler::tryDisassembleSection(ELFSectionRef sectionRef, unsigned s
       continue;
     }
 
+    while (!instSeq.empty() && instSeq.front().offset < offset)
+      instSeq.erase(instSeq.begin());
```

Comment:
I don't see any insertions into this vector at this point in the function, are there any?